spec/rubocop/cop/rspec/rails/travel_around_spec.rb

```diff
@@ -67,6 +67,56 @@
     end
   end
 
+  context 'with `freeze_time` in `around` and other `before` with `{}' do
+    it 'registers offense' do
+      expect_offense(<<~RUBY)
+        before { do_some_preparation }
+
+        around do |example|
+          freeze_time do
+          ^^^^^^^^^^^^^^ Prefer to travel in `before` rather than `around`.
+            example.run
+          end
+        end
+      RUBY
+
+      expect_correction(<<~RUBY)
+        before { do_some_preparation;freeze_time }
```

Comment:
memo)With other automatic corrections for RuboCop offenses, the code is formatted as follows.

### Code
```ruby
# before
# frozen_string_literal: true

before { do_some_preparation;freeze_time }

# after
# frozen_string_literal: true

before do
  do_some_preparation
  freeze_time
end
```

### Log
```
❯ bundle exec rubocop -A test.rb                                 
Inspecting 1 file
C

Offenses:

test.rb:3:8: C: [Corrected] Style/BlockDelimiters: Avoid using {...} for multi-line blocks.
before { do_some_preparation
       ^
test.rb:3:9: C: [Corrected] Layout/TrailingWhitespace: Trailing whitespace detected.
before { 
        ^
test.rb:3:10: C: [Corrected] Layout/MultilineBlockLayout: Block body expression is on the same line as the block start.
before { do_some_preparation ...
         ^^^^^^^^^^^^^^^^^^^
test.rb:3:29: C: [Corrected] Layout/SpaceAfterSemicolon: Space missing after semicolon.
before { do_some_preparation;freeze_time }
                            ^
test.rb:3:29: C: [Corrected] Style/Semicolon: Do not use semicolons to terminate expressions.
before { do_some_preparation;freeze_time }
                            ^
test.rb:4:2: C: [Corrected] Layout/IndentationConsistency: Inconsistent indentation detected.
 freeze_time }
 ^^^^^^^^^^^
test.rb:4:14: C: [Corrected] Layout/BlockEndNewline: Expression at 4, 14 should be on its own line.
 freeze_time }
             ^
test.rb:5:10: C: [Corrected] Layout/IndentationConsistency: Inconsistent indentation detected.
         freeze_time
         ^^^^^^^^^^^

1 file inspected, 8 offenses detected, 8 offenses corrected
```